smartcontractkit/external-adapters-js
# Historical average adapter

## Changes

- Added historical average adapter
- Modified CMC's historical endpoint
- Exported types from CMC adapter

## Steps to Test

See integration/e2e tests and examples from historical average adapter

1. Run CMC adapter
2. Run historical-average adapter
3. Send test requests from README or run e2e tests

## Quality Assurance
